MaskRCNN/config.py

```diff
@@ -110,10 +110,9 @@ def __ne__(self, _):
 # schedule -----------------------
 _C.TRAIN.NUM_GPUS = None         # by default, will be set from code
 _C.TRAIN.WEIGHT_DECAY = 1e-4
-_C.TRAIN.BASE_LR = 1e-2  # defined for total batch size=8. Otherwise it will be adjusted automatically
-_C.TRAIN.WARMUP = 1000   # in terms of iterations. This is not affected by #GPUs
-_C.TRAIN.WARMUP_INIT_LR = 1e-2 * 0.33  # defined for total batch size=8. Otherwise it will be adjusted automatically
-_C.TRAIN.STEPS_PER_EPOCH = 500
+_C.TRAIN.BASE_LR = 1e-2  / 8. / 8. # defined for total batch size=1. It will be adjusted automatically using linear scaling.
```

Comment:
Why double division?

Reply:
Because we made two changes each of which required us to divide the lr by 8. 
1. Changed the BASE_LR to be defined for bs=1 instead of bs=8
2. Removed the code in the optimizer that divided the lr by 8.